R/dl_build_bin.R

```diff
@@ -74,7 +82,15 @@ dl_build_bin <- function(sync=FALSE, GRAN.dir = './GRAN'){
 	}
 	
 	#Once done, write PACKAGES file, use default pkgType for this platform (mac/win)
+	currentListPath <- "./granCurrent.tsv"
```

Comment:
This should be pointing to the GRAN directory. It can't live in the repo as it will just be clobbered on Jenkins.